salt/apps/ocw/templates/webhook-publish.sh.jinja

```diff
@@ -40,33 +47,21 @@ else
     rm $retry_file
 fi
 
-log_message "Pulling ocw-to-hugo"
-cd /opt/ocw/ocw-to-hugo
-git pull || error_and_exit "Can not 'git pull' ocw-to-hugo"
-npm install . || error_and_exit "Can not 'npm install' ocw-to-hugo"
+cd /opt/ocw/hugo-course-publisher \
+    || error_and_exit "Can not cd to hugo-course-publisher!"
 
 log_message "Pulling hugo-course-publisher"
-cd /opt/ocw/hugo-course-publisher
 git pull || error_and_exit "Can not pull hugo-course-publisher"
 yarn install --pure-lockfile \
     || error_and_exit "Can not install hugo-course-publisher"
 
-log_message "Pulling source data"
-aws s3 sync s3://$SOURCE_DATA_BUCKET/ $SOURCE_DATA_DIR --delete --only-show-errors
-if [ $? -ne 0 ]; then
-    error_and_exit "Failed to pull source data"
-fi
-
-log_message "Running ocw-to-hugo"
-cd /opt/ocw/ocw-to-hugo
-node src/bin/index.js -i $SOURCE_DATA_DIR -o $MARKDOWN_DIR \
-    --strips3 --staticPrefix /coursemedia
+log_message "Pulling ocw-to-hugo and source data"
```

Comment:
A little bit of a nitpick, but technically `ocw-to-hugo` is pulled in during the `yarn install` command above.  This command just executes it based on the environment variables.